site/docs/flink.md

```diff
@@ -301,7 +301,22 @@ Table create commands support the most commonly used [flink create clauses](http
 * `COMMENT 'table document'` to set a table description.
 * `WITH ('key'='value', ...)` to set [table configuration](./configuration.md) which will be stored in apache iceberg table properties.
 
-Currently, it does not support computed column, primary key and watermark definition etc.
+Currently, it does not support computed column and watermark definition etc.
+
+### `PRIMARY KEY`
+
+To declare primary key(s) in a table, use `PRIMARY KEY ... NOT ENFORCED`:
+
+```sql
+CREATE TABLE `hive_catalog`.`default`.`sample` (
+    id BIGINT COMMENT 'unique id',
+    data STRING,
+    PRIMARY KEY (id) NOT ENFORCED
```

Comment:
is `id BIGINT PRIMARY KEY NOT ENFORCED COMMENT 'unique id',`  support?

Reply:
Yep, should I add another example ?

Reply:
I don't personally think it's worth another example. My understanding is that this is more of the differences in available FlinkSQL syntax, correct?

If so, I don't think we need to demonstrate all possible forms for the same thing. It would be better left to their documentation to show various forms. Too many examples can make the document hard to read as well.

Reply:
Fine